dart-impl/mpi/src/dart_mpi_op.c

```diff
@@ -133,7 +135,7 @@ static void dart__mpi__op_invoke_custom(
 {
   struct dart_operation_struct *dart_op = get_op(*dptr_);
   DART_LOG_TRACE("Invoking custom operation %p (op=%p, ud=%p)",
-                 dart_op, dart_op->op, dart_op->user_data);
+                 (const void *)dart_op, (const void *)(uintptr_t)dart_op->op, dart_op->user_data);
```

Comment:
Why do we need the intermediate cast to `uintptr_t`?

Reply:
Ahh I see, it might be better then to consistently print `dart_op_t` as `%ld` (it is an integer after all, right?)

Reply:
Agree. Since we define `dart_operation_t` we can print it as `%ld`. What I do not really like is that we often log opaque MPI types as well and do not really know that.